APNS/APNS/APNSPusher.swift

```diff
@@ -15,6 +15,7 @@ public protocol APNSPushable {
                       priority: Int,
                       collapseID: String?,
                       inSandbox sandbox: Bool,
+											liveActivity: Bool,
```

Comment:
@KittenYang Please run `swiftlint autocorrect --config .swiftlint.yml --format` to fix these indentation issues.

Reply:
Done!

Reply:
Thanks for your work. I'll check on my side to fix this issue.